anchor/routes/variables.php

```diff
@@ -94,18 +94,13 @@
         
         $input['key'] = 'custom_' . slug($input['key'], '_');
         
-        foreach ($input as $key => &$value) {
+        foreach ($input as $k => &$value) {
             $value = eq($value);
         }
         
         $validator = new Validator($input);
         
         $validator->add('valid_key', function ($str) use ($key) {
```

Comment:
Either we can drop `use ($key)`, or change the logic that follows to `if($str != $key)` because if we're editing the variable, we don't want a change to happen, thus if `$str != $key` then a change has occurred.